ansible/roles/prometheus/tasks/main.yaml

```diff
@@ -14,3 +14,12 @@
     chart_ref: "kube-prometheus"
     chart_version: "{{ seldon_monitoring_prometheus_operator_chart_version }}"
     values: "{{ seldon_monitoring_prometheus_operator_values }}"
+
+
+- name: Install Prometheus Adapter
```

Comment:
we'll also want to install the config that Seldon uses, similar to here: `https://docs.seldon.ai/seldon-core-2/kubernetes/hpa-rps-autoscaling#installing-and-configuring-the-prometheus-adapter`. Again, we can take this as a follow-up task (the config map would probably end up having the name `seldon-prometheus-adapter-prometheus-adapter` given the name chosen below for the helm release. In the docs I've called it `hpa-metrics` to avoid the duplication.

Reply:
ok will add the config as well.